docs/titan-basics.md

```diff
@@ -239,14 +239,3 @@ The Pearl Expert Touch Wing can only be used with the [Pearl Expert](about-the-c
 console and should be connected before the console is powered up.
 
 
-Controlling Dimmers and Fixtures
```

Comment:
How come this text has been removed?

Reply:
It is in the wrong place. It is actually the start of the next chapter "Controlling Fixtures"